scripts/west_commands/run_common.py

```diff
@@ -6,16 +6,19 @@
 '''
 
 import argparse
-from os import getcwd, path
+import logging
+from os import close, getcwd, path
 from subprocess import CalledProcessError
+import tempfile
 import textwrap
+import traceback
 
 from west import cmake
```

Comment:
You are still importing a bunch of modules from west, so how is this "dewestizing" the runners, which depend on `run_common.py`?

Reply:
The runners do not depend on run_common.py; the opposite is true.